extensions/runtimes/src/index.ts

```diff
@@ -452,33 +452,6 @@ export default async (composer: any): Promise<void> => {
       // return the location of the build artifiacts
       return publishFolder;
     },
-    eject: async (project, localDisk: IFileStorage, isReplace: boolean) => {
```

Comment:
is this not the `eject` implementation for the current runtime?

Reply:
no this is the one in the new runtime definition